src/app/actions/registerHandler.ts

```diff
@@ -0,0 +1,12 @@
+import { Keypair } from "stellar-sdk";
+import setKeypair from "./setKeypair";
+import { ISetKeyPair } from "../types/types";
+
+function registerHandler({ setSecret, setPublicKey }: ISetKeyPair) {
+  // Call create random pair of keys
```

Comment:
If our code is legible and well-developed is not necessary to add comments. The code should speak for us, we already know by reading the pair variable that we're generating random keypairs.

Reply:
Issue addressed in commit 46392d0adecd8be079ef4945707510a632e02f81, which removes unnecessary comments.